test/bin/scenario.sh

```diff
@@ -878,6 +878,8 @@ configure_vm_firewall() {
     # - Incoming for NodePort services
     run_command_on_vm "${vmname}" "sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/tcp"
     run_command_on_vm "${vmname}" "sudo firewall-cmd --permanent --zone=public --add-port=30000-32767/udp"
+    # - Default Prometheus exporter port (for observability and telemetry RF tests)
```

Comment:
Does telemetry use the exporter?

Reply:
You can configure a [Prometheus Exporter in OTEL config file](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter) instead of a Remote Write Exporter. Prometheus exporter will only expose metrics in an endpoint to be able to be consumed.

Reply:
fix comment in https://github.com/openshift/microshift/pull/4825/commits/6cc07b490741146915f03ee1729ef14c3360a1e4 commit